docs/source/requirements.txt

```diff
@@ -1 +1 @@
-# This requirements.txt is intentionally left blank for RTD build
```

Comment:
the path here is `docs/source/requirements.txt`. I'd prefer to move this file to `docs/requirements.txt`.